automated install/basic-install.sh

```diff
@@ -371,6 +420,9 @@ os_check() {
             printf "             %bsudo PIHOLE_SKIP_OS_CHECK=true pihole -r%b\\n" "${COL_LIGHT_GREEN}" "${COL_NC}"
             printf "           (In this case, your previous run of pihole -up will have already updated the local repository)\\n"
             printf "\\n"
+            printf "      If you are running Alpine Linux, ensure abuild is installed:\\n"
```

Comment:
Would this be better displayed only if Alpine had been identified, rather than no matter the distro?

Reply:
Is `abuild` available to be installed from package manager? 

If so shouldn't it be added to the `pihole-meta` package? 

Not an alpine Linux user so I'm not sure what is actually required for `abuild` to work properly.